.travis.yml

```diff
@@ -31,7 +31,7 @@ jobs:
   include:
     - stage: Push bundle and index images to Quay ()
       #if: branch = master AND type = push
-      if: pull_request
+      if: type = pull_request
```

Comment:
why this change?

Reply:
Originally, the job is for "on push-to-master" and the for PR is for testing purposes... before this is merged, I'll change it back to push-to-master

Reply:
and this particuar commit is fixing a typo (missing `type = ` in the condition)